### PR TITLE
fixes bug in ordering of cmds

### PR DESCRIPTION
**Pull request by cramja**
Minor bugfixes to the release scripts. Previously I was saving the wrong directories to environment variables. Also, adding grep to one of the commands makes it work predictably.

### CODE REVIEW

**Comment by cramja** [comment]
@hbdeshmukh This one will be good to have in before the next release candidate.

**Comment by hbdeshmukh** [comment]
Are there any issues with Travis? None of the recent PRs have been scheduled yet. I would like to merge the three PRs related to RC6 without waiting for the Travis to complete. Partly because the changes are small and are verified by @cramja on his laptop and also because they are necessary for the preparation towards release. Any objections?

**Comment by hbdeshmukh** [comment]
@cramja Can you please rebase your branch with the master? I can merge it.

**Comment by cramja** [comment]
I have no objections ;)

File: release/release_cmds.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ create_artifacts() {
 
   export RELEASE_DIR=`pwd`
   cd ..
+  export BASE_DIR=`pwd`
```
Comment (cramja) [inline release/release_cmds.sh]: `$BASE_DIR` is the quickstep root. release dir is `$BASE_DIR/release`

**Pull request merged**